client/src/components/geneExpression/gene.tsx

```diff
@@ -272,8 +296,14 @@ class Gene extends React.Component<Props, State> {
             />
           </div>
         </div>
-        {/* @ts-expect-error ts-migrate(2769) FIXME: No overload matches this call. */}
-        {geneIsExpanded && <HistogramBrush isUserDefined field={gene} />}
+        {geneIsExpanded && (
+          <HistogramBrush
+            // @ts-expect-error ts-migrate(2769) FIXME: No overload matches this call.
+            isUserDefined
+            field={gene}
+            onGeneExpressionComplete={() => {}}
```

Comment:
why the noop?

Reply:
These histograms only exist once a gene is expanded, but I wanted to just keep track of the mini histograms (calling onGeneExpressionComplete on load of each mini histogram, until they're all loaded). TS unfortunately requires I define onGeneExpressionComplete, so I put a empty function in there. Is that okay?